tests/integration_tests/functional/test_api.py

```diff
@@ -749,11 +749,9 @@ def test_drive_patch(test_microvm_with_api):
     fs_vub = drive_tools.FilesystemFile(
         os.path.join(test_microvm.fsfiles, "scratch_vub")
     )
-    vhost_user_socket = "/vub.socket"
+
     # Launching vhost-user-block backend
-    _backend = spawn_vhost_user_backend(
-        test_microvm, fs_vub.path, vhost_user_socket, False
-    )
+    vhost_user_socket = test_microvm.spawn_vhost_user_blk_backend(fs_vub.path, False)
     test_microvm.add_vhost_user_drive("scratch_vub", vhost_user_socket)
```

Comment:
I had this idea too, but the following stopped me (perhaps unjustifiably):
- we have tests with vhost-user as a root device and it requires some special treatment
- we have a test where we want to stop the backend independently and see if FC is still alive. This may not be transparent with that API.
- we have a test with multiple puts that may require additional handling on the uvm side. If we use the drive_id as the socket name, this would require stopping the previous backend first, which is not the same as FC disconnects from one socket and connects to another.